Lib/TagManager/ExampleMediaPlatformAuthManager.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
Filename is ExampleMediaPlatformAuthManager.php, aanpassen naar WebmasterToolsAuthManager.php